examples/src/main/scala/org/apache/spark/examples/ml/FPGrowthExample.scala

```diff
@@ -64,4 +64,3 @@ object FPGrowthExample {
     spark.stop()
   }
 }
-// scalastyle:on println
```

Comment:
@dongjoon-hyun sorry, I missed the ```// scalastyle:off println```
Is it OK with you if  I remove ```// scalastyle:off println``` too?  Since ```println``` is not used in the example

Reply:
Of course, sure!

Reply:
yes, println is not used